akka-projection-eventsourced/src/main/scala/akka/projection/eventsourced/javadsl/EventSourcedProvider.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.eventsourced.javadsl
+
+import java.util.Optional
+
+import akka.NotUsed
+import akka.actor.ClassicActorSystemProvider
+import akka.persistence.query.javadsl.EventsByTagQuery
+import akka.persistence.query.{ NoOffset, Offset, PersistenceQuery }
+import akka.projection.eventsourced.EventEnvelope
+import akka.projection.javadsl.SourceProvider
+import akka.stream.javadsl.Source
+import com.typesafe.config.{ Config, ConfigFactory }
+
+object EventSourcedProvider {
```

Comment:
added already a javadsl counter-part. It will be good to start think on Java API as well.